docs/using/event-types.md

```diff
@@ -47,81 +47,90 @@ have a more specific definition :)
 ### Compatibility modes
 
 Compatibility modes are used to control schema changes. Each mode
-solves a specific problem and thus presents different constraints. It
-ranges from being very constraining, allowing just some minor changes,
-to allowing everything.
+solves a specific problem and thus presents different constraints.
 
-By default, Nakadi uses the forward mode, which is further explained
-below, together with the two other modes: compatible and none.
+Each compatibility mode supports a different set of changes, which are
+described in detail bellow.
 
```

Comment:
editorial proposal:
Nakadi supports different compatibility modes for event types used to control valid schema changes and to ensure schema consistency of published event data. Compatibility modes define restrictions for event schema evolution, i.e. a set cascade of allowed schema changes and thereby different compatibility guarantees for event consumers.